inc/cleantalk-settings.php

```diff
@@ -1155,20 +1165,20 @@ function apbct_settings__display()
     if ( ! $apbct->white_label ) {
         echo '<div style="float: right; padding: 15px 15px 5px 15px; font-size: 13px; position: relative; background: #f1f1f1;">';
 
-        echo __('CleanTalk\'s tech support:', 'cleantalk-spam-protect')
+        echo $apbct->data['wl_brandname_short'] . __('\'s tech support:', 'cleantalk-spam-protect')
             . '&nbsp;'
-            . '<a target="_blank" href="https://wordpress.org/support/plugin/cleantalk-spam-protect">WordPress.org</a>.'
+            . '<a target="_blank" href="' . $apbct->data['wl_support_url'] . '">WordPress.org</a>.'
             . '<br>';
         echo __('Plugin Homepage at', 'cleantalk-spam-protect') .
-             ' <a href="https://cleantalk.org" target="_blank">cleantalk.org</a>.<br/>';
+             ' <a href="' . $apbct->data['wl_url'] . '" target="_blank">' . $apbct->data['wl_url'] . '</a>.<br/>';
         echo '<a href="https://cleantalk.org/publicoffer#cleantalk_gdpr_compliance" target="_blank">'
              . __('GDPR compliance', 'cleantalk-spam-protect')
              . '</a><br/>';
-        echo __('Use s@cleantalk.org to test plugin in any WordPress form.', 'cleantalk-spam-protect') . '<br>';
-        echo __('CleanTalk is registered Trademark. All rights reserved.', 'cleantalk-spam-protect') . '<br/>';
+        echo __('Use stop_email@example.com to test plugin in any WordPress form.', 'cleantalk-spam-protect') . '<br>';
+        echo $apbct->data['wl_brandname_short'] . __(' is registered Trademark. All rights reserved.', 'cleantalk-spam-protect') . '<br/>';
         if ( $apbct->key_is_ok ) {
             echo '<b style="display: inline-block; margin-top: 10px;">' . sprintf(
-                __('Do you like CleanTalk? %sPost your feedback here%s.', 'cleantalk-spam-protect'),
+                __('Do you like ' . $apbct->data['wl_brandname_short'] . '? %sPost your feedback here%s.', 'cleantalk-spam-protect'),
                 '<a href="https://wordpress.org/support/plugin/cleantalk-spam-protect/reviews/#new-post" target="_blank">',
```

Comment:
Does WL dashboard logic support feedback link replacement? If so, we should replace it too, if not, just remove this block.

Reply:
+1

Reply:
There already checked $apbct->white_label mode, on 1155 line.